index.php

```diff
@@ -3,6 +3,11 @@
 
 require_once( 'vendor/autoload.php' );
 
+use Dotenv\Dotenv as Dotenv;
+
+$dotenv = Dotenv::createImmutable( __DIR__ );
+$dotenv->load();
```

Comment:
This works, too, and I think it looks better, but it's not the syntax from the docs so I wonder if it would be appropriate?

```suggestion
Dotenv::createImmutable( __DIR__ )->load();
```

Reply:
This is great!

One thing you could do to remove the need for the `use` statement above:

```php
Dotenv\Dotenv::createImmutable( __DIR__ )->load();
```

Reply:
Yep this is perfect!